doc/nrf/ug_thread_configuring.rst

```diff
@@ -367,7 +367,14 @@ Updating pre-built OpenThread libraries
 You can update nrfxlib's :ref:`nrfxlib:ot_libs` when using any Thread sample if you configure the sample to build OpenThread stack from source with :option:`CONFIG_OPENTHREAD_SOURCES`.
 Use this functionality for example for :ref:`certification <ug_thread_cert>` of your configuration of OpenThread libraries.
 
-After a sample is built, run the following command to overwrite the nrfxlib libraries to your latest version:
+Run the following command to overwrite the nrfxlib libraries to your latest debug version:
+
+.. parsed-literal::
+   :class: highlight
+
+   west build -t install_openthread_libraries
```

Comment:
I guess this target name should be different from the next one, right?

Also add body text to the commit.

Reply:
Yea, copy/paste error.

Well, this commit will be squashed anyway, right?